<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ogrenciIslemleri</class>
 <widget class="QDialog" name="ogrenciIslemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="ogrenciler">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>371</width>
     <height>521</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>185</number>
   </attribute>
  </widget>
  <widget class="QLabel" name="org_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tüm Öğrenciler</string>
   </property>
  </widget>
  <widget class="QLabel" name="baslik">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>631</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { color : red; }</string>
   </property>
   <property name="text">
    <string>Öğrenci İşlemleri</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="no">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Öğrenci No:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ad">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Öğrenci Ad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="soyad">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Öğrenci Soyad:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="no_lineEdit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>220</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ad_lineEdit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>310</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="soyad_lineEdit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>380</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="yeni_kayit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>510</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Yeni Kayıt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sil">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>510</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="guncelle">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>590</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Güncelle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
